discord-canary.spec

```diff
@@ -29,15 +29,14 @@ mkdir -p $RPM_BUILD_ROOT/usr/share/applications
 
 cp -r * $RPM_BUILD_ROOT/opt/DiscordCanary/
 ln -sf /opt/DiscordCanary/DiscordCanary $RPM_BUILD_ROOT/%{_bindir}/
-install -m 755 Discord.desktop $RPM_BUILD_ROOT/usr/share/applications/
+desktop-file-validate %{buildroot}/%{_datadir}/applications/Discord.desktop
```

Comment:
Hmm, why replace the install step with desktop-file-validate? I hadn't used that command before, but seems like it only validates the desktop file?

I get the error:
+ desktop-file-validate /home/vverma7/rpmbuild/BUILDROOT/discord-canary-0.0.9-1.fc24.x86_64//usr/share/applications/Discord.desktop
/home/vverma7/rpmbuild/BUILDROOT/discord-canary-0.0.9-1.fc24.x86_64//usr/share/applications/Discord.desktop: file does not exist